update test md file (#928)
AzureAD/microsoft-identity-web: TESTING.md

    `cd C:\`

    `git clone https://github.com/AzureAD/microsoft-identity-web idweb`

2. cd to the root of the repo (for instance `cd C:\idweb`)

3. Set the version of the templates to test.

   `Set ClientSemVer=1.5.1`

> 4. cd to ProjectTemplates and add the client secrets (or your own config file) to the Configuration.json file

Author: ```suggestion
4. In ProjectTemplates open the Configuration.json file and add the client secrets (or your own config file) .
```
